crates/autopilot/src/database/orders.rs

```diff
@@ -119,13 +119,20 @@ impl Postgres {
             .with_label_values(&["update_limit_order_fees"])
             .start_timer();
         let mut ex = self.0.begin().await?;
-        database::orders::update_limit_order_fees(&mut ex, &ByteArray(order_uid.0), &update)
-            .await
-            .context("update_limit_order_fee")?;
-        if let Some(quote) = quote {
-            database::orders::insert_quote_and_update_on_conflict(&mut ex, &quote)
+        let updated_order_uids =
+            database::orders::update_limit_order_fees(&mut ex, order_spec, &update)
                 .await
-                .context("insert_quote_and_update_on_conflict")?;
+                .context("update_limit_order_fee")?;
+        if let Some(quote) = quote {
+            for order_uid in updated_order_uids {
+                let quote = Quote {
+                    order_uid,
+                    ..quote.clone()
+                };
+                database::orders::insert_quote_and_update_on_conflict(&mut ex, &quote)
+                    .await
+                    .context("insert_quote_and_update_on_conflict")?;
```

Comment:
Should we batch this database operation?

Reply:
Makes sense but this is currently not supported by `sqlx` ([issue](https://github.com/launchbadge/sqlx/issues/294)) 😞

Reply:
I guess they are within a transaction anyways, so the update is atomic. Thanks for checking!